### ISSUE TITLE
Support for BitBox hardware wallet

### ISSUE THREAD

**Issue by rggjan** [enkryptcom/enKrypt#311]
Could you add support for the BitBox02 hardware wallet as well?

MyEtherWallet already supports it...

**Comment by kvhnuke** [comment]
Yes this will be added, however not in the immediate future with the current timeline id estimate atleast 2-3 months

**Comment by ibrahimkhled** [comment]
التعزيز
